recipes/hunter/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "hunter" %}
+{% set version = "3.3.3" %}
+{% set sha256 = "99cddc1f2bbf828232fa37dde939895318f6e451426532d94a90638fc152d3e6" %}
```

Comment:
You can [try grayskull](https://pypi.org/project/grayskull/) to generate these recipes. It will make reviewing them easier. Grayskull won't create a jinja variable for the sha256.